11ty/definitions/master-slave.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Master Slave
+slug: master-slave
+defined: false
+speech: noun
+alt_words:
+  - Main & Secondary
+  - Primary & Secondary
+  - Primary & Replica
+  - Master & Minion
```

Comment:
While I understand the intent behind the master/minion, I wonder if it's better to exclude all master-related terminology.
```suggestion

```